README.md

```diff
@@ -19,123 +19,80 @@ Polyn implements this pattern in a manner that can be applied to multiple progra
 languages, such as Ruby, Elixir, or Python, enabling you to build services that can
 communicate regardless of the language you user.
 
-This documentation describes how Polyn compliant clients should be implemented.
+## Parts of a Polyn Environment
 
-## Conformance
+A Polyn environment is made up of **Applications**, **Nodes**, **Components**, and the
+**Message Bus**. Following is a description of each.
 
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT",
-"RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in
-[RFC2119](https://datatracker.ietf.org/doc/html/rfc2119).
+### Applications
 
-## Definitions
+Polyn applications are processes that implement one or more
+[Components (or Services)](https://www.reactivemanifesto.org/glossary#Component). Ideally,
```

Comment:
I think we should be consistent about calling them `Components` or `Services`. If we call them `Components` in `README.md`, I think `PROTOCOL.md` should do the same.